packages/gestalt/src/Tabs.test.js

```diff
@@ -1,41 +1,45 @@
 // @flow strict
 import { create } from 'react-test-renderer';
-import Tabs from './Tabs.js';
+import Tabs, { TabWithForwardRef } from './Tabs.js';
 
 describe('<Tabs />', () => {
-  test('Only add aria-selected to the active tab', () => {
-    const instance = create(
-      <Tabs
-        tabs={[
-          { text: 'News', href: '#' },
-          { text: 'You', href: '#' },
-          { text: 'Messages', href: '#' },
-        ]}
-        activeTabIndex={0}
-        onChange={() => {}}
-      />,
-    ).root;
-    const links = instance.findAll((element) => element.type === 'a');
-    expect(links && links[0].props['aria-selected']).toEqual(true);
-    expect(links && links[1].props['aria-selected']).toEqual(false);
-    expect(links && links[2].props['aria-selected']).toEqual(false);
-  });
+  // TODO: we no longer support this, but we should
```

Comment:
@dangerismycat we should create a JIRA task

Reply:
Good call! https://jira.pinadmin.com/browse/PDS-2876